easycv/models/detection/utils/misc.py

```diff
@@ -0,0 +1,204 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from functools import partial
+from typing import List, Optional
+
+import numpy as np
+import torch
+import torchvision
+from packaging import version
+from six.moves import map, zip
+from torch import Tensor
+
+if version.parse(torchvision.__version__) < version.parse('0.7'):
+    from torchvision.ops import _new_empty_tensor
+    from torchvision.ops.misc import _output_size
+
+
+def multi_apply(func, *args, **kwargs):
+    """Apply function to a list of arguments.
+    Note:
+        This function applies the ``func`` to multiple inputs and
+        map the multiple outputs of the ``func`` into different
+        list. Each list contains the same type of outputs corresponding
+        to different inputs.
+    Args:
+        func (Function): A function that will be applied to a list of
+            arguments
+    Returns:
+        tuple(list): A tuple containing multiple list, each list contains \
+            a kind of returned results by the function
+    """
+    pfunc = partial(func, **kwargs) if kwargs else func
+    map_results = map(pfunc, *args)
+    return tuple(map(list, zip(*map_results)))
+
+
+@torch.no_grad()
+def accuracy(output, target, topk=(1, )):
```

Comment:
where should these be used, if we need accuracy, we should add it to evaluators

Reply:
detr series debug is used to print logs

Reply:
Used when the DETR series prints logs